### PR TITLE
NEW can define different mysql server host name and port

### PR DESCRIPTION
**Pull request by hregis**

### CODE REVIEW

File: scripts/action_deploy_undeploy.sh [hunk 1]
```diff
@@ -151,6 +151,18 @@ export webSSLCertificateCRT=with.sellyoursaas.com.crt
 export webSSLCertificateKEY=with.sellyoursaas.com.key
 export webSSLCertificateIntermediate=with.sellyoursaas.com-intermediate.crt
 
+# possibility to change the path of sellyoursass directory
+olddoldataroot=`grep 'olddoldataroot=' /etc/sellyoursaas.conf | cut -d '=' -f 2`
```
Comment (eldy) [inline scripts/action_deploy_undeploy.sh:155]: What is goal of this 2 new options ? Can you describe your use case ?

Comment (hregis) [reply scripts/action_deploy_undeploy.sh:155]: @hregis my DOL_DATA_ROOT is not the same in master server and in the instances servers
i use virtualmin for master server and the path is to big to reproduce in instance server

File: scripts/action_deploy_undeploy.sh [hunk 2]
```diff
@@ -204,12 +224,35 @@ echo "CRONHEAD = $CRONHEAD"
...
+fi
+
+echo "Search admin database credential for deployement server in /etc/sellyoursaas.conf"
+dbadminuser=`grep 'databaseuserdeployment=' /etc/sellyoursaas.conf | cut -d '=' -f 2`
+if [[ "x$dbadminuser" == "x" ]]; then
+	dbadminuser="sellyoursaas"
+fi 
+dbadminpass=`grep 'databasepassdeployment=' /etc/sellyoursaas.conf | cut -d '=' -f 2`
+if [[ "x$dbadminpass" == "x" ]]; then
+	echo Failed to get password for mysql admin user
```
Comment (eldy) [inline scripts/action_deploy_undeploy.sh:244]: To not break compatibility, we should here set $dbadminpass to 
`grep 'databasepass=' /etc/sellyoursaas.conf | cut -d '=' -f 2`
if we enter in this if.

So if databasepassdeployment is not into conf file, we take the old key that was databasepass ?
Am i right ?

Comment (hregis) [reply scripts/action_deploy_undeploy.sh]: @eldy yes you right, it's ready

**Pull request merged**